cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -243,7 +247,24 @@ def test_dynamic_field_on_refresh_button():
             3.
             4. dynamic field shouldn't be blank
     """
-    pass
+    sd, ele_label = import_dialog
+    labels = {'var_1 - copied': 'var_2', 'var_2 - copied': 'var_3',
+              'merged': 'var_2_var_3'}
+    catalog_item = appliance.collections.catalog_items.create(
+        appliance.collections.catalog_items.GENERIC,
+        name=fauxfactory.gen_alpha(),
+        description=fauxfactory.gen_alpha(),
+        display_in=True,
+        catalog=catalog,
+        dialog=sd)
+    request.addfinalizer(catalog_item.delete_if_exists)
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    view = navigate_to(service_catalogs, "Order")
+    for key, ele_name in labels.items():
```

Comment:
if we separate out proper names like `label` and `ele_name` from `lable_element_map` it will be easy to understand refresh logic.